utils/service_discovery/sd_docker_backend.py

```diff
@@ -131,11 +131,17 @@ def _get_kube_config(self, c_id, key):
                 if c_id == status.get('containerID', '').split('//')[-1]:
                     return pod.get(key, {})
 
+    def _get_config_id(self, container):
+        config_id = container.get('Labels', {}).get('datadog_id')
```

Comment:
If the first `get` call uses the fallback value, the second call here is going to throw a KeyError